static/serialization_preserve.rs

```diff
@@ -52,11 +52,11 @@ pub trait DeserializeEmbeddedGroup {
 #[inline]
 fn sz_max(sz: cbor_event::Sz) -> u64 {
     match sz {
-        Sz::Inline => 23u64,
```

Comment:
also for consistency since most generated code did `cbor_event::Sz` so I just made these be like that